src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample4_Logging.cs

```diff
@@ -65,7 +65,10 @@ public void EnableEvents(EventLevel level)
 
         protected override void OnEventWritten(EventWrittenEventArgs eventData)
         {
+/* Issue https://github.com/Azure/azure-sdk-for-net/issues/5654 test skipped for net461 */
```

Comment:
It looks like you filed one issue for all the tests being disabled. In general we should file an issue per test library but given there is a small number of issues here we can keep it as one follow-up issue for now, but we may want to split it out by feature area at a future point because the work will most likely be done by different people.

Reply:
Oh, I see, I will split them up.